src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/model/Email.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.model;
+
+import java.util.Map;
+
+public class Email {
+
+    public static final class Builder {
+        private String appId;
+        private String messageId;
+        private String messageType;
+        private Map<String, Object> data;
+        private String emailAddress;
+        private String createdAt;
+
+        public Builder() {
+        }
+
+        public Builder appId(String val) {
+            appId = val;
+            return this;
+        }
+
+        public Builder messageId(String val) {
+            messageId = val;
+            return this;
+        }
+
+        public Builder messageType(String val) {
+            messageType = val;
+            return this;
+        }
+
+        public Builder data(Map<String, Object> val) {
+            data = val;
+            return this;
+        }
+
+        public Builder emailAddress(String val) {
+            emailAddress = val;
+            return this;
+        }
+
+        public Builder createdAt(String val) {
+            createdAt = val;
+            return this;
+        }
+
+        public Email build() {
+            return new Email(this);
+        }
+    }
+    
+    private final String appId;
```

Comment:
i'm not sure what appId is, could it be something like originatingAppId ( i assume)

Reply:
I believe this is how it is set up within the chs infrastructure.

Reply:
Looking at ptf, the kafka client needs to put a field called "app_id" on the message, but we can name this whatever.
So will change to originatingAppId so at a glance is clearer what is happening